src/components/AnonymousChat/BaseChannelItem.tsx

```diff
@@ -81,7 +69,7 @@
             unreadCount={unreadCount}
             isMe={isMe}
             hasFollowButton={hasFollowButton}
-            isFollowing={isFollowing}
+            isFollowing={showFollowingButton}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The renaming of `isFollowing` to `showFollowingButton` makes the code more readable by making it clear that this variable controls the visibility of the follow button. However, make sure that all instances of `isFollowing` are replaced with `showFollowingButton` throughout the codebase to maintain consistency.
```diff
- isFollowing={isFollowing}
+ isFollowing={showFollowingButton}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->